test/run-drun/ok/bad-actor.comp-ref.ok

```diff
@@ -0,0 +1,4 @@
+bad-actor.mo:3.1-4.13: type error [M0141], move these declarations into the body of the main actor or actor class
+bad-actor.mo:11.1-12.13: type error [M0141], move these declarations into the body of the main actor or actor class
+bad-actor.mo:6.1-8.2: type error [M0141], an actor or actor class must be the only non-imported declaration in a program
+  (This is a limitation of the current version and flag -ref-system-api.)
```

Comment:
Just a detail, not worth adjusting, maybe something for backlog: Is it really a limitation or maybe it is the clean way of how we encourage it for the future?

Reply:
Yeah, it seems weird to allow declaration of state outside the actor - maybe we could relax in future and only allow static declarations in the prelude.